_data/sites.json

```diff
@@ -6384,6 +6396,16 @@
         ]
     },
 
+    {
+        "name": "Revolut",
+        "url": "https://www.revolut.com/help/profile/i-want-to-close-my-revolut-account",
+        "difficulty": "easy",
+        "notes": "Open the Revolut app on your phone, go to the Settings tab, press 'Close account' button and follow the instructions.",
```

Comment:
This should mention that they continue to store data for 6 years to comply with regulatory requirements.

Reply:
Something went wrong. The commits for Revolut and GOG were already merged in a pull request I made a month or two ago. Only GitHub commit should be included in this pull request, but I don’t know how to fix that. I did a GitHub commit on a separate branch on my fork and I not longer have the old branch for the rest of the changes, but they still showed up.

Reply:
You should rebase on the upstream master then

Reply:
> You should rebase on the upstream master then

Ok, I’ll do it tomorrow.